sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -511,24 +550,13 @@ export class StreamingReceiver extends MessageReceiver {
 
       // shall retry forever at an interval of 15 seconds if the error is a retryable error
       // else bail out when the error is not retryable or the operation succeeds.
-      const config: RetryConfig<void> = {
-        operation: () =>
-          this.init(
-            // provide a new name to the link while re-connecting it. This ensures that
-            // the service does not send an error stating that the link is still open.
-            true
-          ).then(() => {
-            this._receiverHelper.addCredit(this.maxConcurrentCalls);
-            return;
-          }),
-        connectionId: connectionId,
-        operationType: RetryOperationType.receiverLink,
-        retryOptions: this._retryOptions,
-        connectionHost: this._context.config.host
-      };
-      // Attempt to reconnect. If a non-retryable error is encountered,
-      // retry will throw and the error will surface to the user's error handler.
-      await retry<void>(config);
+      await this.init({
```

Comment:
This is guarded by `shouldOpen` flag which will be false for non retryable errors. Shouldnt this logic be updated too?

Reply:
I will certainly check! :)

Reply:
I updated it by mostly clearing it out - we don't need to distinguish between different kinds of errors as much so I feel a bit better about just logging them and passing them onto the user's error handler.

With that said, I had to adjust a test in streamingReceiver.spec.ts because we actually do get an error in the browser associated with the .idle() trickery we use. It seems legitimate and it's related to websockets so I don't think it's bad, just one area I had to make an adjustment.